### PR TITLE
Updated Global Nav

### PR DESCRIPTION
**Pull request by mateoclarke**
#124 
cityofaustin/techstack#106

Adding in a utility from Bootstrap v4 to help us with responsive display properties:
https://getbootstrap.com/docs/4.0/utilities/display/

TODO:
- [x] Clarify what themes and topics should be visible. Items 2 & 3 in [this description](https://github.com/cityofaustin/techstack/issues/106) seem to conflict.
   - [x] update links
   - [x] make a 'coming soon' placeholder page
- [ ] Double check a11y tab through behavior. 
   - Check markup, add aria tags?
   - skip nav
- [ ] Styling for active page, (need to confirm this in the design)
- [ ] Fix positioning of far right nav list so it doesn't fall of the screen

mar-07-2018 02-43-16

mar-07-2018 02-03-09

### CODE REVIEW

**Comment by mateoclarke** [comment]
There's still some work to do but this PR is getting big so I'll take on the rest of my todo list in a new branch so I can get y'alls review. 🙏

File: src/__tmpdata/navigation.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
+{
+  "themes": [
```
Comment (benweatherman) [inline src/__tmpdata/navigation.json]: Would it make sense to structure this more like a graphql response? That might mean less changes once we put this in the CMS. The `allTopics` response is probably the best starting point for a structure like that.

```
{
  "data": {
    "allThemes": {
      "edges": [
        {
          "node": {
            "title": "womp womp",
            "slug": "womp-womp"
```

etc

File: src/js/page_sections/Navmenu.js [hunk 2]
```diff
@@ -1,115 +1,164 @@
...
       this.focusOnClose();
     }
   }
 
-  getOverlayClassName = () => {
-    let className = `coa-Navmenu__overlay`;
-    if (this.props.isOpen) {
-      className = `${className} ${className}--open`;
+  toggleMobileSublist = (e, openSectionId) => {
+    if (window.innerWidth < this.DESKTOP_BREAKPOINT) {
```
Comment (benweatherman) [inline src/js/page_sections/Navmenu.js]: Does something like this work for SSR? Do we care about that now?

Comment (mateoclarke) [reply src/js/page_sections/Navmenu.js]: hmm, interesting point, but this function only executes on a click event, so I don't think we need to expect this to be node/server-safe.

And if it works for React-Static, then I don't see why it wouldn't work in SSR. And it currently works in the PR deploy. 

But good eye to think about using the `window` with care. I wasn't thinking about that when I wrote this.

**Pull request merged**